app/templates/ratemypdf_stats.html

```diff
@@ -0,0 +1,262 @@
+    <div class="score-position-chart__bar score-position-chart__bar--red" style="width: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    <div class="score-position-chart__arrow score-position-chart__arrow--red" style="left: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    {% elif val > mean - std_dev %}
+    <div class="score-position-chart__bar score-position-chart__bar--yellow" style="width: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    <div class="score-position-chart__arrow score-position-chart__arrow--red" style="left: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>    
+    {% else %}
+    <div class="score-position-chart__bar score-position-chart__bar--green" style="width: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    <div class="score-position-chart__arrow score-position-chart__arrow--green" style="left: {{ percent_of_2_stddev(val, mean, std_dev) }}%;"></div>
+    {% endif %}
+    <div class="score-position-chart__label score-position-chart__label--score">Score: 75</div>
+    <div class="score-position-chart__label score-position-chart__label--median">Median: 70</div>
+    <div class="score-position-chart__label score-position-chart__label--mean">Mean: 68</div>
+    <div class="score-text">{{ val | format_number }}</div>
+  </div>
+</div>
+{% endmacro %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>PDF Statistics for {{ title }}</title>
+  <!-- Bootstrap 5 CSS -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css">
+  <!-- Custom CSS -->
+  <link rel="stylesheet" href="{{ url_for('static', filename='css/ratemypdf.css') }}">
+  <!-- Favicon -->
+  <link rel="icon" type="image/svg+xml" href="{{ url_for('static', filename='img/checkmark-hand-drawn-outline.svg') }}">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js" integrity="sha512-H+LK6+LavIeTpg6vZwJUZf6I1H6XbNfuvxvquyO+LrG+wAR8BHQWbJwvZTzTwsQ+iTFA9d9t4IZ4ef1OdpJ2hw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="icon" type="image/svg+xml" href="{{ url_for('static', filename='img/checkmark-hand-drawn-outline.svg') }}">
+  <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+  <script type="text/javascript">
+    google.charts.load('current', {'packages':['gauge']});
+    google.charts.setOnLoadCallback(drawChart);
+
+    function drawChart() {
+      var data = google.visualization.arrayToDataTable([
+        ['Label', 'Value'],
+        ['Score', {{ complexity_score }}] // Replace with the actual form score
+      ]);
+
+      var options = {
```

Comment:
Overall mixed about the gauge; I really like the idea and it has some cool features, but the aesthetic seems off, especially since it's the main focus of the page. It's fine for now though.